types/client-api.interface.ts

```diff
@@ -0,0 +1,49 @@
+interface APIOptions {
```

Comment:
All type definition files should end with `.d.ts` so it is clear that they are not a compilable source code.

Reply:
@arutkowski00 initially I wanted to do it but these types are not exported and are only used for imports in the main index.d.ts file. Do you think it still requires have to be `.d.ts` files?